pypasta/Repository/Mbox.py

```diff
@@ -184,7 +184,8 @@ def load_index(self, f_index):
         self.index = dict()
         entries = load_file(f_index, must_exist=False)
 
-        for (date, message_id, location) in entries:
+        for entry in entries:
+            date, message_id, location = (entry[0], entry[1], entry[2])
```

Comment:
This will be required, as if we have a patchwork id column, we will have 4 columns in an entry

Reply:
Yes but not for the moment.

Even later, we can do it easier. E.g., sth. like:

for entry in entries:
  for field in entry
     ...

Then we're independant of the size of the tuple.

Reply:
Hmm, for 3 or 4 items in a tuple looping through it seems like overkill

Reply:
The alternative would be?

Reply:
>  date, message_id, location = (entry[0], entry[1], entry[2])

This?

Reply:
And if there's a fourth entry in case of patchwork? We would - conditionally - have to maintain that list. We can avoid that with a simple generalisation.

Reply:
In any case the consumer needs to be aware of what columns of it needs. For 3-4 I find simple tuple destructuring better. If there are more columns than that, then yes, looping. Or maybe array slicing